marketplace-client/src/marketplace/src/app/account/register-stuff/register-stuff.component.html

```diff
@@ -0,0 +1,122 @@
+<div class="card m-3">
+  <h1 class="text-center card-header">Register stuff member</h1>
+  <div *ngIf="!registered" class="card-body">
+    <form [formGroup]="form" (ngSubmit)="onSubmit()">
+      <div class="form-row">
+        <div class="form-group col-6">
+          <label>First Name*</label>
+          <input
+            type="text"
+            formControlName="name"
+            class="form-control"
+            [ngClass]="{ 'is-invalid': submitted && getForm.name.errors }"
+          />
+          <div *ngIf="submitted && getForm.name.errors" class="invalid-feedback">
+            <div *ngIf="getForm.name.errors.required">First Name is required</div>
```

Comment:
what if `submitted && getForm.name.errors` is true, but `getForm.name.errors.required`? empty div? better do it in a single div